src/__tests__/views/LoginPage.test.js

```diff
@@ -9,6 +9,7 @@ import { Provider } from 'react-redux';
 import thunk from 'redux-thunk';
 import { Login } from '../../views/LoginPage';
 import Header from '../../components/header/Index';
+// import Header from '../../components/header/Header';
```

Comment:
@chingsley please remove this if not in use.

Reply:
Removed.